### PR TITLE
W-13743162 work around for viewing image library jh

### PR DESCRIPTION
**Pull request by JennyHajee**

### CODE REVIEW

File: modules/ROOT/pages/troubleshooting.adoc [hunk 1]
```diff
@@ -5,9 +5,32 @@ endif::[]
...
-== Cannot Access the API Management Page to Disconnect Portal
-From the API Management, Settings page, you can disconnect your portal. If you cannot access the Settings page, you can disconnect the portal using the following Curl commands to unlink your Anypoint and Salesforce organization. You can run the commands in a terminal or using Postman and other applications.
+== Cannot See Images in the Builder Preview That Were Uploaded When Creating the Portal
+In the Builder preview, you cannot see the images that were uploaded when creating the portal because the shared user is not a member of any libraries.
+
+To add the shared user as a member of the asset library:
+
+. Log in to the Salesforce organization as a System Administrator.
+. Click the *App Launcher*, enter *files* in the search, and select *Files*. 
+. Click *Libraries*.
```
Comment (jingli) [inline modules/ROOT/pages/troubleshooting.adoc]: Would be clearer to say "Click *Libraries*. in the left navigation"?

File: modules/ROOT/pages/troubleshooting.adoc [hunk 2]
```diff
@@ -5,9 +5,32 @@ endif::[]
...
+. Click *Asset Library* in the list.
+. Click the down arrow and select *Manage Members*.
```
Comment (jingli) [inline modules/ROOT/pages/troubleshooting.adoc]: There is no down arrow for "Manage Members"? It just appears as a button

File: modules/ROOT/pages/troubleshooting.adoc [hunk 3]
```diff
@@ -5,9 +5,32 @@ endif::[]
...
+In the Builder preview, you cannot see the images that were uploaded when creating the portal because the shared user is not a member of any libraries.
+
+To add the shared user as a member of the asset library:
+
+. Log in to the Salesforce organization as a System Administrator.
+. Click the *App Launcher*, enter *files* in the search, and select *Files*. 
+. Click *Libraries*.
+. Click *Asset Library* in the list.
+. Click the down arrow and select *Manage Members*. 
+. In the *Manage Library Members* page, click the down arrow and select *People*.
```
Comment (jingli) [inline modules/ROOT/pages/troubleshooting.adoc]: the "Manage Library Members" is a modal.
Clarity that the down arrow is next to the stack icon

File: modules/ROOT/pages/troubleshooting.adoc [hunk 4]
```diff
@@ -5,9 +5,32 @@ endif::[]
...
++
+The *Add Members* field shows the *search people* box.
+. Click in the *search people* box and select *Builder Preview*.
+. In *Access*, select *Author*. 
+. Click *Add* to allow the Builder Preview user to upload images in the Builder for later use.
++
+The *Current Members* section shows the *Builder Preview* member and *Author* access.
+. Close the page.
++
+. Confirm that you can see the library images in the Builder.
```
Comment (jingli) [inline modules/ROOT/pages/troubleshooting.adoc]: Specify that the Builder User should confirm and not the currently logged in user who is the admin

File: modules/ROOT/pages/troubleshooting.adoc (hunk 1 above)
Comment (JennyHajee) [reply modules/ROOT/pages/troubleshooting.adoc]: For accessibility reasons, we cannot use directional text like up, down, right, and left. I can change the step to:
Click **Libraries** in the navigation pane.

File: modules/ROOT/pages/troubleshooting.adoc [hunk 5]
```diff
@@ -5,9 +5,34 @@ endif::[]
...
 
-== Cannot Access the API Management Page to Disconnect Portal
-From the API Management, Settings page, you can disconnect your portal. If you cannot access the Settings page, you can disconnect the portal using the following Curl commands to unlink your Anypoint and Salesforce organization. You can run the commands in a terminal or using Postman and other applications.
+== Cannot See Images in the Builder Preview That Were Uploaded When Creating the Portal
+In the Builder preview, you cannot see the images that were uploaded when creating the portal because the shared user is not a member of any libraries.
+
+To add the shared user as a member of the asset library:
 
+. Log in to the Salesforce organization as a System Administrator.
+. Click the *App Launcher*, enter *files* in the search, and select *Files*.
```
Comment (Crispy-Salesforce) [inline modules/ROOT/pages/troubleshooting.adoc:14]: Global: As per our style guide, we should use "click" for buttons and "select" for user actions in the UI (See "select", "choose", and "click" at https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference).

**Review by Crispy-Salesforce — APPROVED** [review]

File: modules/ROOT/pages/troubleshooting.adoc (hunk 5 above)
Comment (JennyHajee) [reply modules/ROOT/pages/troubleshooting.adoc:14]: Thanks, Cris. The app launcher is a button/icon.

**Pull request merged**